src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs

```diff
@@ -106,6 +106,13 @@ protected void LogEventsFromTextOutput (string singleLine, MessageImportance mes
 					line = int.Parse (match.Groups ["line"].Value) + 1;
 				var level = match.Groups ["level"].Value.ToLowerInvariant ();
 				var message = match.Groups ["message"].Value;
+
+				// Handle the following which is NOT an error :(
+				// W/ResourceType(23681): For resource 0x0101053d, entry index(1341) is beyond type entryCount(733)
+				if (file.StartsWith ("W/")) {
```

Comment:
Does this need to be `StartsWith("W/", StringComparison.OrdinalIgnoreCase)`?

It might not matter.